lib/embulk/input/mixpanel_api/client.rb

```diff
@@ -118,6 +119,11 @@ def request(params, &block)
             end
           end
           handle_error(response, error_response)
+          if !buf.empty?
```

Comment:
just want to know some backgrounds here: is the response using JSONL (partly)? Meaning it's JSONL but one record could be on multiple lines?

Reply:
It's using JSONL but since we are consuming data by chunk, the last line may contain incomplete JSON